# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../dump/com_err.patch
# Copyright (C) 2009 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./MCONFIG.in.orig	2009-11-20 17:56:43.000000000 +0100
+++ ./MCONFIG.in	2009-11-20 17:56:54.000000000 +0100
@@ -40,7 +40,7 @@
 # indicate where the ext2fs library can be found (this is not needed if you
 # have run `make install-libs' in the e2fsprogs source directory).
 #GLIBDIR=	-L/usr/src/e2fsprogs-0.5c/lib
-GLIBS=		$(GLIBDIR) -L../compat/lib -lcompat -lext2fs -lcom_err
+GLIBS=		$(GLIBDIR) -L../compat/lib -lcompat -lext2fs -lcom_err -lpthread
 
 #
 # Definitions (don't change them unless you know what you are doing)
